internal/contributebot/process_test.go

```diff
@@ -57,12 +88,27 @@ func TestProcessIssueEvent(t *testing.T) {
 			for i, label := range tc.labels {
 				lbls[i] = github.Label{Name: &label}
 			}
+			if tc.title == "" {
+				tc.title = defaultTitle
+			}
 			iss := &github.Issue{
 				Labels: lbls,
+				Title:  github.String(tc.title),
+			}
+			var chg *github.EditChange
+			if tc.action == "edited" {
+				chg = &github.EditChange{}
+				if tc.prevTitle != "" {
+					title := struct {
+						From *string `json:"from,omitempty"`
+					}{From: github.String(tc.prevTitle)}
+					chg.Title = &title
+				}
```

Comment:
Optional: I'm a bit wary of this test-specific language of describing `issueData` drifting from the actual data structure over time, since then you have to understand the logic here in order to understand the actual input. However, I see the benefit in making the fields that the test cares about explicit. A few possibilities:

- Write tests for the abstract structure to the specific data structure.
- Just use the GitHub-provided data structure directly in the test cases.

It's not too bad right now, but thought I'd pass along the suggestion.

Reply:
OK. Let's see how it goes. It may be the case that eventually we are mucking with enough of the fields that it makes more sense to just create the entire struct for each testcase. Can change this later if needed.